Assets/Shopify/UIToolkit/Theme Controllers/ThemeControllerBase.cs

```diff
@@ -65,13 +65,36 @@ public abstract class ThemeControllerBase : MonoBehaviour, IShopCredentials {
         public ShopifyClient Client {
             get {
                 if (_cachedClient == null) {
-                    _cachedClient = new ShopifyClient(LoaderProvider.GetLoader(AccessToken, ShopDomain));
+                    SetupClientAndCart();
                 }
                 return _cachedClient;
             }
         }
         private ShopifyClient _cachedClient;
 
+        public CartController Cart {
```

Comment:
Dont think its nescesary right now but its neat that in the future we could make CartController implement ICartController and allow dependencies to be injected here

Reply:
Yeah. Agreed. Also we should make a `IThemeController`